Directory.Packages.props

```diff
@@ -0,0 +1,47 @@
+<Project>
+  <!-- <Import Project="eng\Versions.props" /> -->
+  
+  <PropertyGroup>
+    <MicrosoftBuildVersion>15.3.409</MicrosoftBuildVersion>
+    <MicrosoftExtensionsVersion>5.0.0-preview.7.20364.11</MicrosoftExtensionsVersion>
+    <!-- In order tests against the same version of NuGet as the SDK. We have to set this to match. -->
+    <NuGetVersion>5.8.0-preview.3.6823</NuGetVersion>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageVersion Include="BenchmarkDotNet.Annotations" Version="0.12.1" />
```

Comment:
Should these be updates not includes?

Reply:
nope the default targets already take `PackageVersion` metadata and update `PackageReference` versions.